cmd/toolbox.cpp

```diff
@@ -616,7 +616,7 @@ void do_compact(db::EnvConfig& config, const std::string& work_dir, bool replace
 }
 
 void do_copy(db::EnvConfig& src_config, const std::string& target_dir, bool create, bool noempty,
-             std::vector<std::string>& names, std::vector<std::string>& xnames, bool dry) {
+             std::vector<std::string>& names, std::vector<std::string>& xnames) {
```

Comment:
Can `std::vector<std::string>& names` and `std::vector<std::string>& xnames` be const parameters?